### PR TITLE
Make gpcopy test go faster, by using a smaller data set.

### PR DESCRIPTION
**Pull request by hlinnaka**
The tests test the ON SEGMENT and PROGRAM options, and the test data size
doesn't seem very important for those. Even the small data set is large
enough that you will have more than a few rows on every segment. This
shaves about 5 seconds off the test on my laptop; every little helps.

### CODE REVIEW

File: src/test/regress/input/gpcopy.source [hunk 1]
```diff
@@ -935,7 +935,7 @@ CREATE TABLE LINEITEM_3 (LIKE LINEITEM);
 CREATE TABLE LINEITEM_4 (LIKE LINEITEM);
 CREATE TABLE LINEITEM_5 (LIKE LINEITEM);
 
-COPY LINEITEM FROM '@abs_srcdir@/data/lineitem.csv' WITH DELIMITER '|' CSV;
+COPY LINEITEM FROM '@abs_srcdir@/data/lineitem_small.csv' WITH DELIMITER '|' CSV;
```
Comment (danielgustafsson) [inline src/test/regress/input/gpcopy.source]: The fault injection test at the end of this file would need to be updated for this.

Comment (asimrp) [reply src/test/regress/input/gpcopy.source]: I cannot find any fault injection test in this file, can you please help me find it?

Comment (hlinnaka) [reply src/test/regress/input/gpcopy.source]: It's down at the very bottom.

**Comment by pivotal-mike** [comment]
Or pipeline failure due to flake. Given its been approved LGTM

**Comment by hlinnaka** [comment]
I ran some quick tests with this smaller data set, to check how many lines are inserted in the lineitem_aborted table before the fault injection cancels it. Seems to be between 2300-2800 lines, and the (small) table only has 2985 lines. I don't think that's enough margin. If we reduce the data set, we would probably get false failures every now and then, if the query isn't cancelled fast enough.

So, I'm closing this PR without pushing.

**Pull request closed without merging**